addons/web/static/lib/daterangepicker/daterangepicker.js

```diff
@@ -278,6 +278,12 @@
         if (typeof options.alwaysShowCalendars === 'boolean')
             this.alwaysShowCalendars = options.alwaysShowCalendars;
 
+        if (typeof options.inputEmpty === 'boolean')
```

Comment:
Not 100% sure if that's clean code, maybe we can put it as one object that contains both the information for if the input should be empty and its placeholder 🤔